File: server/src/main/java/ru/andrey/sensor/temperaturesensor/controller/request/TemperatureResponse.java
package ru.andrey.sensor.temperaturesensor.controller.request;

import lombok.Builder;
import lombok.Data;

import java.time.Instant;

@Data
@Builder
public class TemperatureResponse {

    private double lat;
    private double lon;
    private double temperature;
    private Instant time;
}

File: server/src/main/java/ru/andrey/sensor/temperaturesensor/controller/response/TemperatureResponse.kt
package ru.andrey.sensor.temperaturesensor.controller.response

import org.springframework.hateoas.ResourceSupport
import org.springframework.hateoas.core.Relation
import java.time.Instant

@Relation(value = "temperature", collectionRelation = "temperatures")
data class TemperatureResponse(
        val lat: Double,
        val lon: Double,
        val temperature: Double,
        val time: Instant
) : ResourceSupport()
